### PR TITLE
RSDK-6122: getAccuracy

### PR DESCRIPTION
**Pull request by susmitaSanyal**
Updating getAccuracy return type. 

Scope doc: [here](https://docs.google.com/document/d/1dbVI9njfMWXWg8B_wfRU9vCzi6pGQsgdwgqqjMB-Uhc/edit#heading=h.tcicyojyqi6c)

### CODE REVIEW

**Comment by susmitaSanyal** [comment]
@benjirewis & @stuqdog , PR is ready for review. Let me know if I'm missing anything here.

**Comment by benjirewis** [comment]
Ooh sorry missed this; I feel like I'm not really qualified to be reviewing these flutter PRs 😢 , gonna tag in @njooma.

File: lib/src/components/movement_sensor/movement_sensor.dart [hunk 1]
```diff
@@ -10,6 +10,16 @@ class Position {
...
 
+class Accuracy {
+  Map<String, double> accuracyMap;
+  double hdop;
+  double vdop;
+  int nmeaFix;
+  double compassDegreeError;
+
+  Accuracy(this.accuracyMap, this.hdop, this.vdop, this.nmeaFix, this.compassDegreeError);
+}
```
Comment (njooma) [inline lib/src/components/movement_sensor/movement_sensor.dart:21]: Similar to python, this should be a typealias. You can basically copy the [next line](https://github.com/viamrobotics/viam-flutter-sdk/pull/170/files#diff-9829cc355197d082e394a03aab4e61e2f8e3397abff5c1c4e00f60c67f45debfL13) for example

**Review by njooma — CHANGES_REQUESTED** [review]

File: lib/src/components/movement_sensor/service.dart [hunk 2]
```diff
@@ -38,13 +38,11 @@ class MovementSensorService extends MovementSensorServiceBase {
...
     final movementSensor = _fromManager(request.name);
     final accuracyM = await movementSensor.accuracy(extra: request.extra.toMap());
-    return GetAccuracyResponse(
-      accuracy: accuracyM.accuracyMap,
-      positionHdop: accuracyM.hdop,
-      positionVdop: accuracyM.vdop,
-      positionNmeaGgaFix: accuracyM.nmeaFix,
-      compassDegreesError: accuracyM.compassDegreeError,
-    );
+    return GetAccuracyResponse(accuracy: accuracyM.accuracy)
```
Comment (njooma) [inline lib/src/components/movement_sensor/service.dart:41]: I think you should just be able to `return await movementSensor.accuracy(...)` here

**Review by njooma — APPROVED** [review]

**Pull request merged**